script/test

```diff
@@ -5,12 +5,16 @@ PROJECT_PATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )"
 ROVER="/usr/local/openresty/luajit/bin/rover"
 
 function run_busted_tests {
+    sudo cp -R /tmp/lua_modules .
```

Comment:
I think that this would be faster using `rsync` so it would merge the differences something like `rsync -az  /tmp/lua_modules .`

Reply:
Not sure it's worth in this case, the time spent copying the files is _nothing_ compared to the time spent running the integration tests.

Reply:
ok fair enough